platforms/software/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -202,6 +204,18 @@ public Property<String> getJavaVersion() {
         return javaVersion;
     }
 
+    /**
+     * The directory of the generated project, defaults to the directory the project is generated in.
+     *
+     * @since 8.10
+     */
+    @OutputDirectory
+    @Incubating
+    @Option(option = "project-directory", description = "Set the project directory.")
```

Comment:
💭 The issue suggests to use `--project-directory`, but Gradle already uses `--project-dir` to specify where to start Gradle. It's easy to mistake one for the other. Maybe something like `--init-directory` or `--init-dir` should be used instead?

Reply:
If you used `--project-dir` today (without this PR's changes), does it behave the same way?